views/appView.js

```diff
@@ -0,0 +1,36 @@
+var AppView = Backbone.View.extend({
+  initialize: function(model){
+    this.model = model;
+    //console.log("Here is my appModel");
+    //console.log(this.model);
+    this.listenTo(this.model.currentVideo, "change", this.render);
+    $("#search").on("click", this.search.bind(this));
+  },
+  events: {
+
+  },
+  search: function(){
+    var query = $("#search-term").val().trim().split(" ").join("+");
```

Comment:
Again, why not grab the change event of the dom element and then use that value.